### ISSUE TITLE
Using "sbt assembly" fails to load atlas

### ISSUE THREAD

**Issue by tommyettinger** [ajhager/libgdx-sbt-project.g8#85]
I'm having a very strange issue with the assembly task and a project I generated with this plugin.  The game project can load a TextureAtlas just fine with `sbt desktop/run`, but if I generate with `sbt assembly` and run the jar, it can't parse the atlas.  Here is [the output of sbt assembly](http://pastebin.com/pjvVaDjd), here is [the runtime error](http://pastebin.com/ky1Qfby9).  I'm running on Windows 7 64-bit, libgdx is version 1.2.0, sbt and g8 are installed as this project's README describes, and my project is here https://github.com/tommyettinger/CommandersUnite .  I needed to tweak the generated IntelliJ IDEA projects; I didn't really understand how they worked at the time, and I doubt I could exactly reproduce the steps I took.  Still, all the files in the .idea project are present on my project's repo (except for workspace.xml, which updates whenever you scroll in a file, so it pollutes git).  The issue is present using sbt alone, so the IntelliJ stuff shouldn't matter, but I don't know how this generator works.  I needed to tweak the android/assets/ folder to be visible from desktop in IntelliJ, and I don't know if that propagated into the SBT project.  Any help would be appreciated; I'm hoping this will be a quick fix that can be applied retroactively to my game, but I'm fully willing to re-generate the project.

**Comment by tommyettinger** [comment]
The atlas is retrieved from the jar, I should say.  

``` scala
val atlas = new TextureAtlas(Gdx.files.internal("pack.atlas"))
```

**Comment by tommyettinger** [comment]
Thanks to nexsoftware on IRC (#libgdx), I have this fixed on my side.  proguard-project.txt needs to be updated to not process enums so that it can parse the enums in TextureAtlases; I used the following code:

```
-keepclassmembers enum * {
    public static **[] values();
    public static ** valueOf(java.lang.String);
}
```

You can close this whenever, but it would probably be a good idea to make that small change to the generated proguard settings.

**Comment by aginiewicz** [comment]
Fixed by #86
